src/app/components/Header/index.jsx

```diff
@@ -9,9 +9,35 @@ const StyledHeader = styled.header`
   padding: ${GEL_SPACING_DBL}px;
 `;
 
+const HiddenText = styled.span`
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+`;
+
 const Header = () => (
   <StyledHeader role="banner" aria-label="BBC News">
-    <a href="https://www.bbc.co.uk/news">BBC News</a>
+    <a href="https://www.bbc.co.uk/news">
+      <svg
```

Comment:
The SVG requires further styling so it can be centred vertically.
<img width="581" alt="screen shot 2018-07-27 at 16 48 17" src="https://user-images.githubusercontent.com/916416/43331108-e9e8f50e-91bc-11e8-9d1a-e789129cfea9.png">